.jshintrc

```diff
@@ -2,5 +2,7 @@
   "esnext": true,        // ES6 syntax support
   "globals": {
     "ko": true
-  }
+  },
+  "node": true,
```

Comment:
As we are using some node globals in our test code, we tell JSHint that we are doing so, so that it does not complain when we type things like "require(...)"